internal/sidekick/api/model.go

```diff
@@ -157,6 +157,8 @@ type API struct {
 	// State contains helpful information that can be used when generating
```

Comment:
Consider splitting the internal/sidekick changes into a separate PR. That way the sidekick team can approve these independent of the entire PR.

Reply:
https://github.com/googleapis/librarian/pull/3078